### DISCUSSION TITLE
Top-Down id Error when Running Model Configuration

### DISCUSSION THREAD

**Discussion by djg9324** [talmolab/sleap#1587] [Help!]
Top-Down Id seems to successfully train the centroid model of a multi-animal labeled file. However, when it reaches to the model configuration training, an error in terminal states "ValueError: Input 0 is incompatible with layer model: expected shape=(None, 64, 64, 1), found shape=(1, 22, 22, 1)." How do I fix or adjust my training parameters so that this error no longer occurs?

**Comment by roomrys** [comment]
Hi @djg9324,

Do you have an input scaling set on the centered instance model? While the centroid model expects (and encourages) an input scaling less than 1, the centered instance model requires the input scaling to be kept at 1.

If that is not the case, it would be helpful to see the output of the terminal starting from `Starting Training/Inference` up until the error.

Thanks,
Liezl
